gensim/models/word2vec.py

```diff
@@ -197,7 +197,7 @@
 import numpy as np
 
 from gensim.utils import keep_vocab_item, call_on_class_only, deprecated
-from gensim.models.keyedvectors import KeyedVectors, pseudorandom_weak_vector
+from gensim.models.keyedvectors import KeyedVectors, pseudorandom_weak_vector, Vocab
```

Comment:
Is this really enough? Can you please add a test that successfully loads an older gensim model?

Reply:
Also, this import appears unused to flake8, so we should add a directive to ignore it.